app/views/followups/_snippets.html.erb

```diff
@@ -7,20 +7,19 @@
   <p><strong>Challenging a refusal?</strong> Here are some snippets that may help:</p>
 <% end %>
 
-<div class="snippet-library">
+<div class="snippet-library js-snippet-library">
```

Comment:
Not quite sure where the best place for this comment is, but looks like we'll need to handle "no snippets for this section" (or, just don't show the section at all)

![Screenshot 2021-01-22 at 09 39 16](https://user-images.githubusercontent.com/282788/105508501-525b6980-5cc4-11eb-955b-62e4c74f757e.png)